internal/xsql/conn.go

```diff
@@ -48,7 +48,7 @@ func (c *Conn) CheckNamedValue(value *driver.NamedValue) (finalErr error) {
 }
 
 func (c *Conn) BeginTx(ctx context.Context, opts driver.TxOptions) (_ driver.Tx, finalErr error) {
-	onDone := trace.DatabaseSQLOnConnBeginTx(c.connector.trace, &ctx,
+	onDone := trace.DatabaseSQLOnConnBegin(c.connector.trace, &ctx,
```

Comment:
лучше в metrics/sql.go сделать фикс типа такого
```go
	t.OnConnBeginTx = func(info trace.DatabaseSQLConnBeginTxStartInfo) func(trace.DatabaseSQLConnBeginTxDoneInfo) {
		start := time.Now()
		if config.Details()&trace.DatabaseSQLTxEvents != 0 {
			return func(info trace.DatabaseSQLConnBeginTxDoneInfo) {
				txBegin.With(map[string]string{
					"status": errorBrief(info.Error),
				}).Inc()
				txBeginLatency.With(nil).Record(time.Since(start))
			}
		}

		return nil
	}
```

Reply:
я специально разделил эти 2 колбэка чтобы считать/мониторить/трейсить вызовы Begin и BeginTx по-отдельности

Reply:
этот фикс все равно нужен, но кажется он не поможет правильно считать метрику ydb_go_sdk_ydb_database_sql_conns